openedx_webhooks/info.py

```diff
@@ -152,6 +153,9 @@ def is_committer_pull_request(pull_request: PrDict) -> bool:
 
     repo = pull_request["base"]["repo"]["full_name"]
     org = repo.partition("/")[0]
+    # Check if this is a named release branch
+    # compare_url is "https://api.github.com/repos/octocat/Hello-World/compare/{base}...{head}"
+    open_release = "open-release" in pull_request["base"]["compare_url"].split("...")[0]
```

Comment:
`pull_request["base"]["ref"]` will be a more direct check. its value is `"open-release/lilac.master"`.